GithubApp/DesignFiles/CheckBox.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  CheckBox.swift
+//  GithubApp
+//
+//  Created by BrainX on 09/06/2022.
+//
+
+import Foundation
+import UIKit
+class CheckBox: UIButton {
+    // Images
+    let checkedImage = UIImage(named: "check")! as UIImage
```

Comment:
`UIImage` is already `UIImage`. so you dont have to cast it. Also remove the force unwraping.